runtime/hydradx/src/xcm.rs

```diff
@@ -223,12 +223,31 @@ mod remove_when_updating_to_stable2412 {
 	}
 }
 
+pub struct RestrictedAssetHubAliases;
+impl Contains<Location> for RestrictedAssetHubAliases {
+	fn contains(target: &Location) -> bool {
+		match target.unpack() {
+			// Allow system parachains under the Polkadot relay
+			(1, [Parachain(id)]) if matches!(id, 1001 | 1002 | 1004 | 1005) => true,
```

Comment:
I would just check if the para_id is less than 2000. All parachains with ids less than that are considered system chains